public-api/vx/notification.php

```diff
@@ -53,6 +52,7 @@
 							$RESPONSE['limit'] = 100;
 					}
 					
+					$RESPONSE['max_read'] = notification_GetLastReadNotification($user_id);
```

Comment:
Does this add `max_read` to the response? Fo which calls in that case?

Reply:
Yes, it's added to all /vx/notification/*/feed calls.